token/client/src/token.rs

```diff
@@ -377,7 +432,7 @@ where
     /// Create and initialize a new token account.
     pub async fn create_auxiliary_token_account(
         &self,
-        account: &S,
+        account: &dyn Signer,
```

Comment:
Whoa, we should've done this the whole time

Reply:
its actually the same thing, trait objects are always dynamically sized. nothing outside of tests uses this but if we need to pass an `Arc` through ever i might see if i can use `AsRef`...